docs/textinput.md

```diff
@@ -105,6 +105,29 @@ Inherits [View Props](view.md#props).
 
 ---
 
+### `accessibilityErrorMessage`
+
+String read by screenreaders to indicate an error state of a TextInput.
+Setting accessibilityInvalid to true activates the error message.
```

Comment:
```suggestion
String read by screenreaders to indicate an error state of a TextInput. 
Setting accessibilityInvalid to true activates the error message. 
Note: This prop models `aria-errormessage` but differs in that it does not provide an element id. 
```

Reply:
I replied in https://github.com/facebook/react-native/pull/33468#discussion_r958455539